server/plugin/api.go

```diff
@@ -1,4 +1,4 @@
-package main
+package plugin
```

Comment:
I opted for `plugin` as it's a general term that can be used in other plugins. Another consideration was `app`, as it's also used in the server.